server/mdm/apple/util.go

```diff
@@ -141,3 +143,15 @@ func secureRandInt(min, max int64) (int, error) {
 	}
 	return random, nil
 }
+
+func FmtErrorChain(chain []mdm.ErrorChain) string {
```

Comment:
this seems to be the exact same function as `*MDMAppleCheckinAndCommandService.fmtErrorChain`, can we completely replace the struct function in favor of this?